File: 2. Managers/MusicAPI.Managers/ViewModels/UserProfile.cs
namespace MusicAPI.Managers.ViewModels
{
    public class UserProfile
    {
        public string Country { get; set; } = string.Empty;

        public string DisplayName { get; set; } = string.Empty;

        public string EmailAddress { get; set; } = string.Empty;

        public ExplicitContentFilter ExplicitContentFilter { get; set; } = new ExplicitContentFilter();

        public ExternalUrl ExternalUrls { get; set; } = new ExternalUrl();

        public Followers Followers { get; set; } = new Followers();

        public string Href { get; set; } = string.Empty;

        public string Id { get; set; } = string.Empty;

        public Image[] Images { get; set; } = [];

        public string Product { get; set; } = string.Empty;

        public string Type { get; set; } = string.Empty;

        public string Uri { get; set; } = string.Empty;
    }
}

File: 1. Clients/angular-showcase/src/app/modules/data-models/userProfile.ts
import { ExplicitContentFilter } from './explicitContentFilter';
import { ExternalUrl } from './externalUrl';
import { Followers } from './followers';
import { Image } from './image';
export class UserProfile {
  constructor() {
    this.country = '';
    this.displayName = '';
    this.emailAddress = '';
    this.explicitContentFilter = new ExplicitContentFilter();
    this.externalUrls = new ExternalUrl();
    this.followers = new Followers();
    this.href = '';
    this.id = '';
    this.images = [];
    this.product = '';
    this.type = '';
    this.uri = '';
  }

  country: string;
  displayName: string;
  emailAddress: string;
  explicitContentFilter: ExplicitContentFilter;
  externalUrls: ExternalUrl;
  followers: Followers;
  href: string;
  id: string;
  images: Image[];
  product: string;
  type: string;
  uri: string;
}
